src/tokens/AssetBalanceFiat.tsx

```diff
@@ -0,0 +1,68 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { LayoutChangeEvent, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { useDollarsToLocalAmount } from 'src/localCurrency/hooks'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { positionsStatusSelector, totalPositionsBalanceUsdSelector } from 'src/positions/selectors'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import {
+  stalePriceSelector,
+  tokenFetchErrorSelector,
+  totalTokenBalanceSelector,
+} from 'src/tokens/selectors'
+
+interface Props {
+  onLayout?: (event: LayoutChangeEvent) => void
+}
+
+const AssetBalanceFiat = ({ onLayout }: Props) => {
+  const { t } = useTranslation()
+
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const totalTokenBalanceLocal = useSelector(totalTokenBalanceSelector)
+  const totalPositionsBalanceUsd = useSelector(totalPositionsBalanceUsdSelector)
+  const totalPositionsBalanceLocal = useDollarsToLocalAmount(totalPositionsBalanceUsd)
+
+  const tokenFetchError = useSelector(tokenFetchErrorSelector)
+  const tokensAreStale = useSelector(stalePriceSelector)
+  const positionsStatus = useSelector(positionsStatusSelector)
+
+  const totalBalanceLocal =
+    totalTokenBalanceLocal || totalPositionsBalanceLocal
+      ? new BigNumber(totalTokenBalanceLocal ?? 0).plus(totalPositionsBalanceLocal ?? 0)
+      : undefined
+
+  const balance = totalBalanceLocal?.toFormat(2) ?? new BigNumber(0).toFormat(2)
+  // TODO handle when fetching asset balances
+  const displayedBalance =
+    tokenFetchError || tokensAreStale || positionsStatus === 'error' ? ' –' : balance
```

Comment:
I think for now we should ignore errors fetching positions and still display what we have.
To match the balance display logic in the home feed.